src/Core/View.js

```diff
@@ -82,15 +114,30 @@ function _preprocessLayer(view, layer, provider) {
     // probably not the best place to do this
     if (layer.type == 'color') {
         layer.frozen = layer.frozen || false;
-        layer.visible = layer.visible == undefined ? true : layer.visible;
-        layer.opacity = layer.opacity == undefined ? 1.0 : layer.opacity;
-        layer.sequence = 0;
+        _defineLayerProperty(view, layer, 'visible', true);
```

Comment:
if `layer.visible == false` the layer is hidden. The object's properties  take precedence over the default value.